article/article.Rmd

```diff
@@ -0,0 +1,224 @@
+
+```{r set, warning=FALSE}
+md_intersect(as_messydate("2001-01-01..2001-01-20"),as_messydate("2001-01"))
+md_union(as_messydate("2001-01-01..2001-01-20"),as_messydate("2001-01"))
+md_multiset(as_messydate("2001-01-01..2001-01-20"),as_messydate("2001-01"))
+```
+
+As well, some arithmetic operations are available for messydates. For instance, one can add or subtract one year to all messy dates in a vector.
+
+```{r operate, warning=FALSE, message=FALSE}
+tibble::tibble(date = dates_expand, add = dates_expand + "1 day", subtract = dates_expand - "1 year")
+```
+
+## Case Study - 2001 Battles
+
+Dates, even for some recent events, can be messy. Take the dates of battles in 2001 according to [Wikipedia](https://en.wikipedia.org/wiki/List_of_battles_in_the_21st_century) included in \pkg{messydates}.
+The dates of these battles are often approximate (i.e. the day in which a battle started or ended is not precise) or come from unreliable sources (i.e. the date is precise but not trustworthy).
+
+```{r data, warning=FALSE, message=FALSE}
+battles <- messydates::battles
+battles
+```
+
+\pkg{messydates} facilitates working with these dates as we can, for example, check which dates are precise, get the median values for imprecise dates and find the longest battle in 2001.
+
+```{r working with dates, warning=FALSE, message=FALSE}
+messydates::is_precise(battles$Date)
+as.Date(battles$Date, median)
+as.numeric(as.Date(battles$Date, max) - as.Date(battles$Date, min))
+```
+
+Getting the timing can be important for researchers, however, when faced with date imprecision, researchers usually have to chose between making arbitrary choices (e.g. adding "-01-01" to all incomplete dates) or work imprecise dates (e.g. keep the year only). Yet, both choices may lead to bias results. This is especially true if researchers' are looking to generate inferences. Let's assume, for instance, certain researcher is interested in the relationship between the United States (US) being a party to a conflict and the duration of the conflict in 2001. The researcher theorizes that conflicts involving the US have a shorter duration because the US has the most powerful military in the world. But this relationship could be mediated by the number of parties involved in the conflict. Using \pkg{messydates}, we can create two different date variables in the battles data to be our dependent variables representing conflict time, one variable with an arbitrary cut of point and the other variable with random values for uncertain or approximate dates. As our independent variable, we create a dummy variable for whether the US was involved in the conflict. As our control, we code the number of actors in the conflict. With these variables we build two linear regression models, one using the arbitrary dates and the other the random dates.
+
```

Comment:
Please put sentences/clauses on new lines so that I can offer sentence/clause level comments